cmd/cadvisor.go

```diff
@@ -124,6 +149,31 @@ func main() {
 	klog.V(1).Infof("enabled metrics: %s", includedMetrics.String())
 	setMaxProcs()
 
+	plugins := map[string]container.Plugin{
+		"containerd": containerd.NewPluginWithOptions(&containerd.Options{ // TODO(@spartan0x117): Make containerdEnvMetadataWhiteList non-global and add to these options
+			ContainerdEndpoint:  *ArgContainerdEndpoint,
+			ContainerdNamespace: *ArgContainerdNamespace,
+		}),
+		"docker": docker.NewPluginWithOptions(&docker.Options{
+			DockerEndpoint: *ArgDockerEndpoint,
+			DockerTLS:      *ArgDockerTLS,
+			DockerCert:     *ArgDockerCert,
+			DockerKey:      *ArgDockerKey,
+			DockerCA:       *ArgDockerCA,
+		}),
+		"crio": crio.NewPlugin(), // TODO(@spartan0x117): Add options
+		"podman": podman.NewPluginWithOptions(
+			*ArgPodmanEndpoint,
+			&docker.Options{
+				DockerEndpoint: *ArgDockerEndpoint,
+				DockerTLS:      *ArgDockerTLS,
+				DockerCert:     *ArgDockerCert,
+				DockerKey:      *ArgDockerKey,
+				DockerCA:       *ArgDockerCA,
+			}),
+		"systemd": systemd.NewPlugin(), // TODO(@spartan0x117): Add options.
```

Comment:
There are no options at present.
```suggestion
		"systemd": systemd.NewPlugin(),
```